modules/pagerduty/settings.go

```diff
@@ -14,6 +14,7 @@ type Settings struct {
 
 	apiKey           string
 	escalationFilter []interface{}
+	scheduleIDs      []interface{}
```

Comment:
Tying into supporting other filters, should these all be top level, or should we have a `filtering` config, which these are under?